[Bug]: Convex cannot store files locally or via S3

### Error Message and Logs

```js
files-fb53fbea5cf529dd.js:1 Error: Uploaded file appears to be HTML content.
    at g (files-fb53fbea5cf529dd.js:1:21761)
    at async Promise.all (index 0)
    at async handleUpload (files-fb53fbea5cf529dd.js:1:22325)
    at async onChange (files-fb53fbea5cf529dd.js:1:8043)
g @ files-fb53fbea5cf529dd.js:1
api/storage/upload?token=0146cec9dcc570a1c3747439a811a0e416d65fef16450f7c2a827937439d00c41ba0bbb802226748ef44304cd4fa5d1b84d2f8939e982f6202:1  Failed to load resource: the server responded with a status of 404 ()
```


### Steps to Reproduce

1. Deploy the Convex template
2. open the dashboard
3. login
4. go to files
5. upload a file for pc
6. look in the console log in the browser
7. where is also a red pop up notification that will appear.

### Example Repository URL

_No response_

### Coolify Version

v4.0.0-beta.462

### Are you using Coolify Cloud?

No (self-hosted)

### Operating System and Version (self-hosted)

Ubunto 24.04

### Additional Information

I tried to upload a file to test before starting to build with my self-hosted Convex. I couldn't upload files to my Convex instance. 

Then I tried to connect S3 storage. I used self-hosted MinIO (RELEASE.2025-04-22T22-12-26Z). I created all of the buckets and got the access key, secret key, and S3 endpoint URL. 

I tested the S3 storage itself and confirmed the S3 configuration is correct—it saw all the buckets—but Convex couldn't see anything. 

It was blind. SQLite works, and I connected a self-hosted PostgreSQL database to Convex and it worked. So only the file storage is broken in the Coolify template file.

<img width="478" height="219" alt="Image" src="https://github.com/user-attachments/assets/62c93765-1fbc-4058-b20b-2ce5c1c3b05e" />

<img width="394" height="103" alt="Image" src="https://github.com/user-attachments/assets/da90e929-eb6c-4821-bf73-84380107a758" />

--- a/templates/compose/convex.yaml
+++ b/templates/compose/convex.yaml
@@ -12,14 +12,15 @@ services:
       - data:/convex/data
     environment:
       - SERVICE_URL_BACKEND_3210
+      - SERVICE_URL_SITE_3211
       - INSTANCE_NAME=${INSTANCE_NAME:-self-hosted-convex}
       - INSTANCE_SECRET=${SERVICE_HEX_64_SECRET}
       - CONVEX_RELEASE_VERSION_DEV=${CONVEX_RELEASE_VERSION_DEV:-}
       - ACTIONS_USER_TIMEOUT_SECS=${ACTIONS_USER_TIMEOUT_SECS:-}
       # URL of the Convex API as accessed by the client/frontend.
-      - CONVEX_CLOUD_ORIGIN=${SERVICE_URL_DASHBOARD}
+      - CONVEX_CLOUD_ORIGIN=${SERVICE_URL_BACKEND}
       # URL of Convex HTTP actions as accessed by the client/frontend.
-      - CONVEX_SITE_ORIGIN=${SERVICE_URL_BACKEND}
+      - CONVEX_SITE_ORIGIN=${SERVICE_URL_SITE}
       - DATABASE_URL=${DATABASE_URL:-}
       - DISABLE_BEACON=${DISABLE_BEACON:?false}
       - REDACT_LOGS_TO_CLIENT=${REDACT_LOGS_TO_CLIENT:?false}

